draft/2021-11-24-this-week-in-rust.md

```diff
@@ -97,25 +97,18 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 ### Rust Compiler Performance Triage
 
-A large amount of noise in the comparisons this week, likely due to new
-probabilistic query hash verification increasing likelihood of changes in each
-benchmark; solutions are being tracked in [rustc-perf#1105].
 
-Otherwise, though, the week largely amounted to a neutral one for performance.
-There were some regressions, particularly in doc builds, as a result of the
-addition of portable SIMD. These are relatively speaking minor and primarily
-impact small crates.
+This week, there were a number of cases where the `incr-unchanged` variants of `inflate` went up or down by 5% to 6%; we believe these are instances of increased noise in benchmarks documented on [rustc-perf#1105](https://github.com/rust-lang/rustc-perf/issues/1105). I was tempted to remove these from the report, but its non-trivial to re-construct the report "as if" some benchmark were omitted.
 
-[rustc-perf#1105]: https://github.com/rust-lang/rustc-perf/issues/1105
+Otherwise, there were some nice wins for performance. For example, PR [#90996](https://github.com/rust-lang/rust/issues/90996) more than halved the compilation time for `full` builds of `diesel` by revising how we hash `ObligationCauseData`. If anyone is interested, it might be good to follow-up on the effects of PR [#90352](https://github.com/rust-lang/rust/issues/90352), "Simplify `for` loop desugar", where we have hypothesized that the increased compilation time is due to more LLVM optimizations being applied.
```

Comment:
Since this actually is a doc benchmark:

```suggestion
Otherwise, there were some nice wins for performance. For example, PR [#90996](https://github.com/rust-lang/rust/issues/90996) more than halved the time to document `full` builds of `diesel` by revising how we hash `ObligationCauseData`. If anyone is interested, it might be good to follow-up on the effects of PR [#90352](https://github.com/rust-lang/rust/issues/90352), "Simplify `for` loop desugar", where we have hypothesized that the increased compilation time is due to more LLVM optimizations being applied.
```